static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -199,7 +214,8 @@ function fetchEventPoints(
   dateRange: [string, string],
   disasterType: string,
   severityFilter?: SeverityFilter,
-  useCache?: boolean
+  useCache?: boolean,
+  eventDisplayProps?: EventDisplayProp[]
```

Comment:
nit: also update function comment with the new param

Reply:
done